lms/static/sass/shared/_course_object.scss

```diff
@@ -7,33 +7,44 @@
       width: flex-grid(4);
       @include margin-right(flex-gutter());
       float: left;
-
-      &:nth-child(3n+3) {
-        @include margin-right(0);
-      }
     }
 
     .courses-listing {
+      @include outer-container;
       @include clearfix();
       margin: 0;
       padding: 0;
       list-style: none;
 
       .courses-listing-item {
-        width: flex-grid(4);
-        @include margin-right(flex-gutter());
-        @include float(left);
+        height: 360px;
```

Comment:
Your assumption was correct. I used it to make sure that all course cards have the same height. I will address this by applying your suggestions.